Neos.Flow/Classes/Http/Headers.php

```diff
@@ -530,7 +530,7 @@ private function prepareValues($headerName, array $values)
     {
         $preparedValues = [];
         foreach ($values as $value) {
-            $preparedValues[] = sprintf("%s: %s", $headerName, $value);
+            $preparedValues[] = sprintf("%s: %s", $headerName, $headerName === 'Authorization' ? '****' : $value);
```

Comment:
Is it safe to scramble it here - where and how is this Headers being used?

Reply:
Right now (IIRC 6.0+) it's not really used anywhere (and marked deprecated). I do plan to reintroduce it for the ActionRequest/Response classes as an abstraction though, since the class generally gives good convenience constructs around some edgy headers like `cache-control`.

Reply:
Mh, I would suggest to omit this part for now (or create a dedicated PR for a newer branch) since it might break code that relies on `Headers::getPreparedValues()` to return the actual header values

Reply:
Convinced. I'll add that change back into 7.x though I'm not sure if that should really be scrambled. See example of the Browser, where we might at some point use the Headers class to render them and scrambling would be an issue.